src/components/VIconButton/VIconButton.vue

```diff
@@ -49,6 +49,14 @@ export default {
      * documentation on `VIcon`.
      */
     iconProps: {},
+    /**
+     * `variant` prop to pass to `VButton`. See `VButton` for documentation
+     * and possible values.
+     */
+    variant: {
+      type: String,
+      default: 'plain',
+    },
```

Comment:
I actually tried that originally when I made the suggestions for this PR but it did not work for some reason. Maybe you can try checking out the branch and getting it to work? The issue I was seeing is that default variant would get used instead of the one being passed through `v-bind` :disappointed: